packages/codebytes/src/helpers/index.ts

```diff
@@ -39,14 +12,8 @@ export const trackClick = (
       target,
     });
   }
-  const options = getOptions();
-  const page_name = options.renderMode
-    ? `${options.clientName}_${options.renderMode}`
-    : options.clientName;
 
   trackUserClick({
```

Comment:
we could also default tracking data to an empty object and then we don't need two branches

Reply:
even if we don't default to an empty object, it should still work, because `...undefined` functions like an empty object

<img width="327" alt="Screen Shot 2022-02-08 at 4 30 12 PM" src="https://user-images.githubusercontent.com/4481281/153078918-67546b39-5b5e-4382-ae87-a80b7a3ff490.png">

so we can do

```
export const trackClick = (
  target: string,
  trackingData?: Omit<UserClickData, 'target'>
) => trackUserClick({...trackingData, target});
```